src/Tribe/Views/V2/Hooks.php

```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
+	 * @param array $sheets Array of sheets to search for.
+	 *
+	 * @return array Modified array of sheets to search for.
+	 */
+	public function customizer_inline_stylesheets( $sheets, $css_template ) {
+		$v2_sheets = [
+			'tribe-events-views-v2-skeleton',
+			'tribe-events-views-v2-full',
+		];
+
+		// Unenqueue legacy sheets.
+		if ( ( $keys = array_keys( $sheets, 'tribe-events-calendar-style' ) ) !== [] ) {
+			foreach( $keys as $key ) {
+				unset( $sheets[$key] );
```

Comment:
* :no_entry_sign: Array keys must be surrounded by spaces unless they contain a string or an integer.
* :no_entry_sign: There must be a single space after the first square bracket
* :no_entry_sign: There must be a single space before the last square bracket

<sub>hash: 307b1461285168303449b10a2c09702a</sub>